x-pack/plugins/actions/server/sub_action_framework/types.ts

```diff
@@ -76,6 +77,36 @@ export type Validators<Config, Secrets> = Array<
   ConfigValidator<Config> | SecretsValidator<Secrets>
 >;
 
+export interface PreSaveConnectorHookParams<Config, Secrets> {
```

Comment:
Can you import these types from `x-pack/plugins/actions/server/types.ts`?

Reply:
Waiting on that to be merged and then will do the changes https://github.com/elastic/kibana/pull/194081